libs/ledger-live-common/src/families/hedera/api/mirror.ts

```diff
@@ -131,7 +131,7 @@ export async function getOperationsForAccount(
       // Set a value just so that it's considered confirmed according to isConfirmedOperation
       blockHeight: 5,
       blockHash: null,
-      extra: {},
+      extra: {consensusTimeStamp: consensus_timestamp,},
```

Comment:
Note that with upcoming PR https://github.com/LedgerHQ/ledger-live/pull/4235, this will soon have to be typed.